runtime/include/chplmemtrack.h

```diff
@@ -29,6 +29,8 @@
 #include <stdlib.h>
 
 
+///// These entry points support debugging.
+
```

Comment:
I think this and the comment below are a mis-characterization of the difference between the two sets of functions. These ones you describe here as supporting debugging are for more than just that. The programmer might have other reasons for being interested about memory usage, either during or at the end of execution. If you really need to to split the functions into two groups, you could say this first set is to support the user interface embodied in the module code, while the second set supports the data collection upon which the first set relies.  I would argue that chpl_reportMemInfo() belongs in the first set, though, since its only purpose is to cause the reporting requested by the various config consts.